### DISCUSSION TITLE
How I can change the color for the buttons for pagination?

### DISCUSSION THREAD

**Discussion by Jero8i** [callstack/react-native-paper#4335] [General]
I want to change the color for the pagination buttons control

**Comment by josuazurbruegg** [comment]
You should be able to override the coloring of the pagination control buttons by setting the corresponding theme property value which seems to be either "text" or "onSurface", depending on your theme version.

The following theme definition worked for me:
```
<DataTable.Pagination
  {...props} // whatever additional property values
  theme={{
    colors: {
      text: '#616161',
      onSurface: '#616161',
      elevation: { level2: '#000' },
    },
  }
/>
```

You might be able to derive further corresponding theme values by having a look at the definitions in the source around here: https://github.com/callstack/react-native-paper/blob/bc0ecb1a7381928ebb5bc10fa491e27046430dc1/src/components/DataTable/DataTablePagination.tsx#L115-L132
